Bottom: e5e0bf93b234df4835ee07dd1f74bb803d28890d
Top:    1f7dd20dc2487bab361e6e6be75b2b6ce41a01e2
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-04-23 14:04:14 -0400

Refresh of mpc5200-dma-split

---

diff --git a/sound/soc/fsl/mpc5200_psc_i2s.c b/sound/soc/fsl/mpc5200_psc_i2s.c
index f93460d..1d30a64 100644
--- a/sound/soc/fsl/mpc5200_psc_i2s.c
+++ b/sound/soc/fsl/mpc5200_psc_i2s.c
@@ -405,6 +405,12 @@ static int __devinit psc_i2s_of_probe(struct of_device *op,
 	if (rc)
 		dev_info(psc_dma->dev, "error creating sysfs files\n");
 
+	rc = snd_soc_register_dais(psc_dma->dai, 1);
+	if (rc != 0) {
+		printk("Failed to register DAI\n");
+		return 0;
+	}
+
 	/* Tell the ASoC OF helpers about it */
 	of_snd_soc_register_cpu_dai(op->node, psc_dma->dai, 1);
